powertools-idempotency/powertools-idempotency-redis/src/main/java/software/amazon/lambda/powertools/idempotency/persistence/redis/Constants.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates.
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package software.amazon.lambda.powertools.idempotency.persistence.redis;
+
+final class Constants {
+    private Constants() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    public static final String REDIS_CLUSTER_MODE = "REDIS_CLUSTER_MODE";
```

Comment:
should we move this to the RedisPersistenceStore config ? 
to have everything at the same place:

```java
RedisPersistenceStore.builder() 
  .withHost(System.getenv("REDIS_HOST")) 
  .withPort(System.getenv("REDIS_PORT")) // optional, default 6379 
  .withUsername(secretsProvider.getValue("REDIS_USERNAME")) 
  .withPassword(secretsProvider.getValue("REDIS_PASSWORD")) 
  .withSsl(true) // optional, default false ? or should it be true as default 
  .withClusterMode(true) // optional, default false
  .build();
```